tools/wasme/cli/Makefile

```diff
@@ -25,6 +25,12 @@ GCFLAGS := all="-N -l"
 GCLOUD_PROJECT_ID?=solo-public
 BUILD_ID := $(BUILD_ID)
 
+ifdef WASME_USE_PODMAN
+	CONTAINERCLI := podman
+else
+	CONTAINERCLI := docker
+endif
```

Comment:
Rather than have `WASME_USE_PODMAN` can we just use `CONTAINER_CLI ?= docker` and allow the user to set it that way. There are more options than docker and podman, correct?

Reply:
i don't know any other options;
podman is docker cli compatible, the other programs are not

Reply:
i think it is better to be restrictive here; to prevent potential confusion